apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -67,10 +73,18 @@ export default function RubricSettings({
           setStatus(STATUS.ERROR);
         } else {
           response.json().then(data => {
+            console.log('1st');
+            console.log(data);
             if (!data.attempted) {
               setStatus(STATUS.NOT_ATTEMPTED);
             } else if (data.lastAttemptEvaluated) {
               setStatus(STATUS.ALREADY_EVALUATED);
+            } else if (data.status === 1000) {
+              setStatus(STATUS.ERROR);
+            } else if (data.status === 1001) {
+              setStatus(STATUS.PII_ERROR);
+            } else if (data.status === 1002) {
```

Comment:
please use the js version of these shared constants, here and below. for example: 
* https://github.com/code-dot-org/code-dot-org/blob/a594ea39f828105724c9d6fb486a2dbb74b6b149/apps/src/templates/rubrics/AiAssessmentBox.jsx#L12
* https://github.com/code-dot-org/code-dot-org/blob/a594ea39f828105724c9d6fb486a2dbb74b6b149/apps/src/templates/rubrics/AiAssessmentBox.jsx#L27

Reply:
fixed